docs/ReferenceGuides/UnderscoredAttributes.md

```diff
@@ -131,6 +131,17 @@ library), instead of at an arbitrary point in time.
 For more details, see the forum post on
 [dynamic method replacement](https://forums.swift.org/t/dynamic-method-replacement/16619).
 
+## `@_eagerMove`
+
+When applied to a value, indicates that the value's lifetime is _not_ lexical,
+that releases of the value may be hoisted without respect to deinit barriers.
+
+When applied to a type, indicates that all values which are instance of that
+type are themselves `@_eagerMove` as above, unless overridden with `@_lexical`.
```

Comment:
We should be clear that `@_eagerMove` on types is only a hint. It means that the values may be eagerly moved, but will be honored only for concrete types. It's not broadly true for all "values which are instances of that type". It only applies to static types

Reply:
Clarified that it only applies to values whose static type is eager move.